include/rgl/api/core.h

```diff
@@ -481,6 +493,25 @@ rgl_node_rays_set_ring_ids(rgl_node_t* node, const int32_t* ring_ids, int32_t ri
 RGL_API rgl_status_t
 rgl_node_rays_transform(rgl_node_t* node, const rgl_mat3x4f* transform);
 
+/**
+ * Creates or modifies VelocityDistortRaysNode.
+ * The node distorts rays according to the sensor's linear velocity and angular velocity.
+ * NOTE! 
+ * The distortion is performed in the sensor-local coordinate frame. 
+ * Considering that, This node has to be placed before TransformRaysNode. 
+ * If not velocity and angular velocity must be provided in the global coordinates frame.
```

Comment:
I don't like this description.

> in the global coordinates frame.

We cannot assume that TransformRaysNode is always transforming to the global coordinate frame.

I'd describe it as:
`The velocities passed to that node must match the coordinate frame in which rays are described/transformed. E.g. ... (here put some examples)`